src/main/resources/avro/variantmethods.avdl

```diff
@@ -70,11 +70,11 @@ record SearchVariantsRequest {
   string variantSetId;
 
   /**
-  Only return variant calls which belong to call sets with these IDs.
+  Only return variant calls which belong to samples with these IDs.
   If an empty array, returns variants without any call objects.
   If null, returns all variant calls.
   */
-  union { null, array<string> } callSetIds = null;
+  union { null, array<string> } sampleNames = null;
```

Comment:
I won't explain the technical reasons – your example is very rare in practice; especially in the final data release, we almost never want to put two calling methods in one VCF/VariantSet. I don't mind if API doesn't work elegantly in this extreme case (note that it still works, just not elegantly). In contrast, the current API model is complicating the most common use cases. It actually doesn't really resolve your example if you simply take VCF and SAM submissions.